integration-tests/gatsby-image/cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "http://localhost:9000",
+  "chromeWebSecurity": false
```

Comment:
Things were erroring out without this, and I think it's fairly harmless?

Reply:
What was the error?

Here's the docs for this for other's reference https://docs.cypress.io/guides/guides/web-security.html#Disabling-Web-Security